tezos/timing/src/schema_stats.sql

```diff
@@ -22,16 +22,22 @@ CREATE TABLE IF NOT EXISTS contexts (
   hash TEXT UNIQUE
 );
 
+CREATE TABLE IF NOT EXISTS keys (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  key TEXT NOT NULL UNIQUE
```

Comment:
@sebastiencs did you notice any performance difference with this implementation? We are missing an index on this one, without it I think things will start to slow down after a bunch of keys have been added

Reply:
My understanding is that `UNIQUE` automatically makes an index on that column.
I tried to add a `CREATE INDEX key_idx ON keys(key);` but it makes a error saying that the index already exist.

Reply:
@sebastiencs oh, in sqlite I think `UNIQUE` already creates an index, let me check.